string cmd
uint8 duration_10ms
